src/chainstate/stacks/boot/pox-2.clar

```diff
@@ -1077,10 +1167,18 @@
           reward-set-indexes: (list),
           first-reward-cycle: first-reward-cycle,
           lock-period: lock-period })
-
+      
       ;; return the lock-up information, so the node can actually carry out the lock. 
-      (ok { stacker: stacker,
-            unlock-burn-height: new-unlock-ht }))))
+      (ok (merge event-info {
+            name: "delegate-stack-extend", 
+            data: {
+                pox-addr: pox-addr, 
+                unlock-burn-height: new-unlock-ht, 
```

Comment:
Similar to the previous comment, the `event-info. burnchain_unlock_height` is "stale" for the few functions that update it, e.g.:
```
balance: 7937301999969877n
burnchain_unlock_height: 115n
data: { extend_count: 1n, unlock_burn_height: 120n }
locked: 2062698000000123n
name: 'stack-extend'
stacker: 'ST11NJTTKGVT6D1HY4NJRVQWMQM7TVAR091EJ8P2Y'
```

The `data.unlock_burn_height` value is correct but `burnchain_unlock_height` is old.